Makefile

```
@@ -462,12 +462,23 @@ dev.shell.%: ## Run a shell on the specified service's container.
 dev.dbshell:
 	docker compose exec mysql80 bash -c "mysql"
 
+dev.dbsetupmysql8:
+	docker compose exec -T mysql80 bash -e -c "mysql -uroot mysql" < provision-mysql80.sql
```

This should work fine without the `bash -c` indirection:
```suggestion
	docker compose exec -T mysql80 mysql -uroot mysql < provision-mysql80.sql
```